### ISSUE TITLE
Classify folders into a taxonomy of models

### ISSUE THREAD

**Issue by AnderBiguri** [CambridgeCIA/LION#66]
Current idea:

- CNNs (just general CNN models)
- Unrolled iterative methods
- Post/pre processing
- Learned Regularizers
- PnP (its a lerned regularized, but implicitly. same folder as previous?)
- Unsupervised methods

**Comment by mbkiss** [comment]
Yes, that is what I have in thr Benchmarking as well 🤓 Although I did not separate CNNs and post-/preprocessing, this makes sense because we might implement methods that do make the domain change 💯

**Comment by AnderBiguri** [comment]
@mbkiss great! I was thinking having CNNs in one folder, and then postprocessing can just import any of these models, if they want. I many of the other methods may want to import a "built in" CNN, so that folder is more like a box of LEGO.
